cpp/src/io/csv/writer_impl.cu

```diff
@@ -251,7 +251,7 @@ struct column_to_strings_fn {
     return cudf::strings::detail::from_timestamps(
       column,
       format,
-      strings_column_view(column_view{data_type{type_id::STRING}, 0, nullptr}),
+      strings_column_view(column_view{data_type{type_id::STRING}, 0, nullptr, nullptr, 0}),
```

Comment:
we don't have a way to create an empty column view?

Reply:
Maybe something like this?
```
strings_column_view(make_empty_column(type_id::STRING)->view())
```